app/Models/CaseStudy.php

```diff
@@ -12,7 +12,7 @@
  * @property array $name
  * @property array $description
  * @property string $photo
- * @property string $slug
+ * @property array $slug
```

Comment:
What is the reason to change it to array? we're not translating slugs

Reply:
fixed, came from previous pull request (before your changes in code)